target-platforms/target-platform-latest/target-platform-latest.target

```diff
@@ -21,6 +15,11 @@
 			<unit id="org.eclipse.sdk.ide" version="0.0.0"/>
 			<repository location="https://download.eclipse.org/eclipse/updates/latest/"/>
 		</location>
+		<location type="InstallableUnit" includeConfigurePhase="true" includeMode="planner" includeSource="true">
+			<repository location="https://download.eclipse.org/eclipse/updates/4.33-P-builds/"/>
```

Comment:
Please don't use 4.33-Pbuilds as they will be gone soon. Rather add the bundles to the orbit location with latest https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/milestone/S202409180633 as used in https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/blob/master/eclipse.platform.releng.prereqs.sdk/eclipse-sdk-prereqs.target

Reply:
OK, done.